cleo_sdk/CLEO_Utils.h

```diff
@@ -31,6 +31,7 @@ namespace CLEO
     OPCODE_READ_PARAM_INT()
     OPCODE_READ_PARAM_UINT()
     OPCODE_READ_PARAM_FLOAT()
+    OPCODE_READ_PARAM_ANY32() // any regular single-slot variable (practically integers and floats)
```

Comment:
not necessarily a variable. could be any 32-bit value

Reply:
Fixed